### DISCUSSION TITLE
[General] Projects API

### DISCUSSION THREAD

**Discussion by hanniabu** [zkSync-Community-Hub/zksync-developers#241] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

Hey, I was curious if there's an API to get the information listed on this page: https://zksync.dappradar.com/ecosystem

### What is the specific issue or error you're encountering?

n/a

### Can you share the error messages or logs you're receiving, if any?

n/a

### Have you made any recent changes to the contract before encountering this issue?

n/a

### Are there any external libraries or contracts that your contract interacts with?

n/a

### Can you provide the relevant portions of your contract code where the issue is occurring?

n/a

### Have you tried to isolate the problem, and if so, what were the results?

n/a

### What steps have you already taken to try to resolve the issue?

n/a

**Comment by MexicanAce** [comment, marked as answer]
Hi @hanniabu 👋 !

The apps/projects page you're referencing is hosted by DappRadar. Here's a link to their API docs : https://api-docs.dappradar.com/

**Comment by inpaslimabj** [comment]
The page you provided appears to list various projects and applications built on zkSync. To retrieve similar information programmatically, you would typically need access to an API provided by zkSync or a third-party service that aggregates data from zkSync. 

You can check zkSync's official documentation or developer resources to see if they offer an API for accessing ecosystem information. Alternatively, you might explore third-party data providers or APIs that specialize in tracking and analyzing blockchain ecosystem data.

**Comment by go4comdathern** [comment]
Looking for an API to access information listed on the zkSync ecosystem page.  Currently exploring available options.

**Comment by lynnrosado** [comment]
You can access the information listed on the page https://zksync.dappradar.com/ecosystem programmatically by utilizing DappRadar's API. You may need to refer to DappRadar's documentation or contact their support for more details on accessing the ecosystem data through their API.

**Comment by tcaniao** [comment]
You can retrieve information about zkSync ecosystem projects using the zkSync API. Check the zkSync documentation or contact their support for details on accessing this data programmatically.

**Comment by crissshawn** [comment]
The user is inquiring about the availability of an API to retrieve information listed on the zksync.dappradar.com/ecosystem page. No specific issue or error message was shared, and there are no recent changes to the contract or interactions with external libraries or contracts. The user has not attempted to isolate the problem or taken any steps to resolve the issue. No repository link was provided.

**Comment by eddiedenna** [comment]
The user is inquiring about an API to retrieve information listed on the zksync.dappradar.com/ecosystem page. They do not provide any error messages or logs, have not made any recent changes to the contract, and are not interacting with any external libraries or contracts. The user has not tried to isolate the problem or taken any steps to resolve the issue. They do not provide a repo link.
